src/wp-includes/class-wp-autoload.php

```diff
@@ -0,0 +1,502 @@
+		'wp_media_list_table'                         => 'wp-admin/includes/class-wp-media-list-table.php',
+		'wp_ms_sites_list_table'                      => 'wp-admin/includes/class-wp-ms-sites-list-table.php',
+		'wp_ms_themes_list_table'                     => 'wp-admin/includes/class-wp-ms-themes-list-table.php',
+		'wp_ms_users_list_table'                      => 'wp-admin/includes/class-wp-ms-users-list-table.php',
+		'wp_plugin_install_list_table'                => 'wp-admin/includes/class-wp-plugin-install-list-table.php',
+		'wp_plugins_list_table'                       => 'wp-admin/includes/class-wp-plugins-list-table.php',
+		'wp_post_comments_list_table'                 => 'wp-admin/includes/class-wp-post-comments-list-table.php',
+		'wp_posts_list_table'                         => 'wp-admin/includes/class-wp-posts-list-table.php',
+		'wp_privacy_data_export_requests_list_table'  => 'wp-admin/includes/class-wp-privacy-data-export-requests-list-table.php',
+		'wp_privacy_data_removal_requests_list_table' => 'wp-admin/includes/class-wp-privacy-data-removal-requests-list-table.php',
+		'wp_privacy_policy_content'                   => 'wp-admin/includes/class-wp-privacy-policy-content.php',
+		'wp_privacy_requests_table'                   => 'wp-admin/includes/class-wp-privacy-requests-table.php',
+		'wp_screen'                                   => 'wp-admin/includes/class-wp-screen.php',
+		'wp_site_health_auto_updates'                 => 'wp-admin/includes/class-wp-site-health-auto-updates.php',
+		'wp_site_health'                              => 'wp-admin/includes/class-wp-site-health.php',
+		'wp_site_icon'                                => 'wp-admin/includes/class-wp-site-icon.php',
+		'wp_terms_list_table'                         => 'wp-admin/includes/class-wp-terms-list-table.php',
+		'wp_theme_install_list_table'                 => 'wp-admin/includes/class-wp-theme-install-list-table.php',
+		'wp_themes_list_table'                        => 'wp-admin/includes/class-wp-themes-list-table.php',
+		'wp_upgrader_skin'                            => 'wp-admin/includes/class-wp-upgrader-skin.php',
+		'wp_upgrader'                                 => 'wp-admin/includes/class-wp-upgrader.php',
+		'wp_users_list_table'                         => 'wp-admin/includes/class-wp-users-list-table.php',
+
+		/* Classes in wp-admin/includes/deprecated.php. */
+		'wp_user_search'                              => 'wp-admin/includes/deprecated.php',
+		'wp_privacy_data_export_requests_table'       => 'wp-admin/includes/deprecated.php',
+		'wp_privacy_data_removal_requests_table'      => 'wp-admin/includes/deprecated.php',
+	);
+
+	/**
+	 * Whether the autoloader has already been registered or not.
+	 *
+	 * Avoid registering the autoloader multiple times.
+	 *
+	 * @static
+	 * @access private
+	 *
+	 * @var bool
+	 */
+	private static $registered = false;
+
+	/**
+	 * Register the autoloader.
```

Comment:
```suggestion
	 * Registers the autoloader.
```
Docblock summaries should use third-person singular verbs (`Registers`, not `Register`). [Ref](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#documenting-tips)

:1234: Applies throughout.